english/new-years/new-years.md

```diff
@@ -1,2 +1,4 @@
 Ah . . . New Year's Eve . . . it all comes back to you now . . . sort of. 
-It was a costume party, and, come to think of it, the Chief Marshmallow was there too.
\ No newline at end of file
+It was a costume party, and, come to think of it, the Chief Marshmallow was there too.
+
+He was obviously angry about something, it could be the horrible smell that was coming from the [raccoon in the corner](raccoon-corner/raccoon-corner.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.